### ISSUE TITLE
Batch size

### ISSUE THREAD

**Issue by edshkim98** [AnTao97/dgcnn.pytorch#23]
Hi, thank you for providing the pytorch version!
I am currently training my model using pytorch dgcnn model, but due to the size of my dataset and model, I am trying to train my model using stochastic gradient descent (batch size = 1).
However, it doesnt seem to be straight forward to implement it due to use of nn.batchnorm1d and I get errors when I set batch size=1.
Is there any solution or work around so that I can train my model using batch size =1?
Thank you

**Comment by antao97** [comment]
If you use bachsize=1,  delete the `nn.batchnorm1d`.
